datasketch/minhash.py

```diff
@@ -131,6 +139,17 @@ def _init_permutations(self, num_perm: int) -> np.ndarray:
     def _parse_hashvalues(self, hashvalues):
         return np.array(hashvalues, dtype=np.uint64)
 
+    def enable_gpu(self) -> None:
```

Comment:
Can we make this a constructor parameter? What do you think?

Reply:
Yes, that could be much better and clean.